src/Core/src/Platform/Android/WindowExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Android.App;
+using Microsoft.Maui.Graphics;
+using AndroidX.Core.View;
+
+namespace Microsoft.Maui
+{
+	public static class WindowExtensions
+	{
+		public static void SetStatusBarColor(this Activity activity, Color color) =>
+			SetStatusBarColor(activity, color, color?.GetLuminosity() < 0.5);
```

Comment:
`color?.GetLuminosity() < 0.5` this won't produce expected result in all scenarios (at least it didn't when I tested it in Xamarin for https://github.com/xamarin/XamarinCommunityToolkit/pull/812). Sometimes text is poorly visible to the eye.

Maybe it's better to use something like `IsDarkForTheEye` (https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Extensions/ColorConversionExtensions.shared.cs#L124) for automatic text color adjustment.